File: Square Patterns/5. Square Odd Numbers/son.cpp
#include <iostream>
using namespace std;
int main(){
    int n;
    cout<<"Enter a Number : ";
    cin>>n;

    for(int i=1; i<=n; i++){
        for(int j=1; j<=n; j++){
            cout<<2*j-1<<' ';
        }
        cout<<'\n';
    }
}

/*
Output

1 3 5 7 9 
1 3 5 7 9 
1 3 5 7 9 
1 3 5 7 9 
1 3 5 7 9 

*/

File: Square Patterns/5. Square Odd Numbers/son.java
import java.util.*;
public class son{
    public static void main(String [] args){
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter a Number : ");
        int n = sc.nextInt();

        for(int i=1; i<=n; i++){
            for(int j=1; j<=n; j++){
                System.out.print(2*j-1 + " ");
            }
            System.out.println();
        }

        sc.close();
    }
}

/*
Output

1 3 5 7 9 
1 3 5 7 9 
1 3 5 7 9 
1 3 5 7 9 
1 3 5 7 9  

*/